.Gruntfile.babel.js

```diff
@@ -210,7 +210,9 @@ module.exports = (grunt) => {
     // https://docs.cypress.io/guides/guides/module-api.html
     const options = {
       run: {
-        headed: grunt.option('browser') === true
+        headed: grunt.option('browser') === true,
+        key: 'q6whky',
+        record: true
```

Comment:
1. This `key` is not the record key, it is the project ID
2. The record key must not be committed to source, [it must be private](https://docs.cypress.io/guides/dashboard/projects.html#Record-key)
3. To make it private, you must fetch it from an environment variable instead, and only insert the two keys (`key` and `record`) IFF the environment variable is defined. Use [this syntax](https://github.com/okTurtles/group-income-simple/blob/ea1f6d1865710039d820616b5fcde37b1c43ad7c/frontend/model/contracts/group.js#L301-L313) to do that.